src/ui/edit-content/content-attributes/BooleanAttributeField.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  fieldInputPropTypes,
+  fieldMetaPropTypes,
+} from 'redux-form';
+
+import RadioInput from 'ui/common/form/RenderRadioInput';
+
+const BooleanAttributeField = ({
+  label,
+  input,
+  meta,
+  ...rest
+}) => {
+  const toggleElements = [
+    { id: 'true', label: 'Yes' },
+    { id: 'false', label: 'No' },
```

Comment:
Just thinking about this. Maybe these labels need to be internationalized too? so, pass `intl` object from parent to this BooleanAttributeField component and instead of label:'Yes', use label: intl.formatMessage({id: 'yes.string.id'}. Just a thought